### ISSUE TITLE
tag a stable release for f-droid

### ISSUE THREAD

**Issue by robinpaulson** [nickw1/Freemap#6]
f-droid is a repository of free software for android and derivatives. it would be great to get a version released there, the guidelines for inclusion are here:
https://f-droid.org/wiki/page/Inclusion_Policy
https://f-droid.org/wiki/page/Inclusion_How-To

cheers,

**Comment by nickw1** [comment]
Yes, adding to f-droid looks a good idea. Sorry for the late reply btw!

**Comment by Poussinou** [comment]
Any news about this?

**Comment by nickw1** [comment]
Yes, sorry - it's been on my to-do list for a while. The previous version  of OpenTrail (0.3l it has its own repository https://github.com/nickw1/opentrail/) can I think be submitted to f-droid, there was a lingering error on shutdown which I never resolved (probably due to the library it uses) but aside from that it is functional. I just need to make the installation instructions clearer as it relies on a load of external open source libraries. Will try and do this soon.

**Comment by nickw1** [comment]
I have now submitted it (the old version of OpenTrail, 0.3)
